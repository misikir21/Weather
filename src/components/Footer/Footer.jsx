import style from './Footer.module.css';

const Footer = () => (
  <footer className={style.footer}>
    <div className={style.footerRow}>
      <p>@Copyright</p>
      <p>Created by Binyam Yohannes</p>
    </div>
  </footer>
);

export default Footer;
