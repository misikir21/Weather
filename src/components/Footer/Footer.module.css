.footer {
  padding: 0.5em 0;
}

.footerRow {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  align-items: center;
  background: white;
  padding: 0 1em;
  font-size: 1.125rem;
  opacity: 0.8;
  height: 30px;
  max-height: 50px;
}

.footerRow p:last-of-type {
  font-size: 1rem;
  opacity: 0.6;
}
