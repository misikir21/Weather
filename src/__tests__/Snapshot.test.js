import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer/Footer';

describe('Snapshot Tests', () => {
  it('renders Footer correctly', () => {
    const footerComponent = renderer.create(<Footer />).toJSON();
    expect(footerComponent).toMatchSnapshot();
  });
});
