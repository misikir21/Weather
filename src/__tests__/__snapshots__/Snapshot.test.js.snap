// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Tests renders Footer correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="footerRow"
  >
    <p>
      @Copyright
    </p>
    <p>
      Created by Binyam Yohannes
    </p>
  </div>
</footer>
`;
